charts/victoria-metrics-distributed/templates/vmauth-read.yaml

```diff
@@ -1,21 +1,25 @@
 {{- if .Values.read.global.vmauth.enabled }}
 {{- $fullname := include "vm.fullname" . }}
-{{- $ctx := dict "helm" . "appKey" (list "read" "global" "vmauth") "fullname" $fullname }}
+{{- $ctx := dict "helm" . "appKey" (list "vmauth" "spec") "fullname" $fullname }}
+{{- if (((.Values).read).global).vmauth }}
```

Comment:
This check is not needed since we checked `{{- if .Values.read.global.vmauth.enabled }}` on line1.